setup.py

```diff
@@ -78,7 +87,12 @@ def readFile(filename):
 	requirements = None
 
 
-conf_dict = read_configuration("""setup.cfg""", ignore_option_errors=True)
+conf_dict = None
+
+
+with warnings.catch_warnings():
+	warnings.simplefilter("ignore")
+	conf_dict = read_configuration("""setup.cfg""", ignore_option_errors=True)
```

Comment:
This is technical debt ... it is ugly code. either revert way back and use simple open function or refactor to use new build system and use the untested cutting edge. 🤷🏻